resources/lib/navigation/directory_search.py

```diff
@@ -91,14 +92,11 @@ def search_add():
             row_id = _search_add_bygenreid(SEARCH_TYPES[type_index], genre_id)
     else:
         raise NotImplementedError('Search type index {} not implemented'.format(type_index))
-    # Execute the research
-    if row_id is None:
-        return False
-    # Redirect to "search" endpoint (otherwise causes problems with Container.Refresh used by context menus)
-    end_of_directory(False)
-    url = common.build_url(['search', 'search', row_id], mode=G.MODE_DIRECTORY)
-    common.container_update(url, False)
-    return True
+    if row_id is not None and search_query(row_id, 0, False):
```

Comment:
keep the deleted comment

Reply:
and also is needed to add a comment with the reason beacuse of container_update call:
in similar way of the deleted:
> Redirect to "search" endpoint (otherwise causes problems with Container.Refresh used by context menus)

in the future you forget, and new developers may not understand the reasons for this recall